core/src/main/java/org/apache/iceberg/AllManifestsTable.java

```diff
@@ -53,8 +53,9 @@
           Types.NestedField.required(11, "contains_nan", Types.BooleanType.get()),
           Types.NestedField.optional(12, "lower_bound", Types.StringType.get()),
           Types.NestedField.optional(13, "upper_bound", Types.StringType.get())
-      )))
-  );
+      ))),
+      Types.NestedField.optional(14, "content", Types.StringType.get())
+      );
```

Comment:
nit: should not change space for unmodified lines.

Reply:
done